src/domain/tokens/entities/schemas/token.schema.ts

```diff
@@ -1,37 +1,16 @@
-import { JSONSchemaType, Schema } from 'ajv';
-import { buildPageSchema } from '@/domain/entities/schemas/page.schema.factory';
-import { Token, TokenType } from '@/domain/tokens/entities/token.entity';
+import { buildZodPageSchema } from '@/domain/entities/schemas/page.schema.factory';
+import { TokenType } from '@/domain/tokens/entities/token.entity';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { z } from 'zod';
 
-export const TOKEN_SCHEMA_ID =
-  'https://safe-client.safe.global/schemas/tokens/token.json';
+export const TokenSchema = z.object({
+  address: AddressSchema,
+  decimals: z.number().nullish().default(null),
+  logoUri: z.string(),
```

Comment:
I would've said that this could be a `z.string().url()` but I see it's just a `string` in AJV. Perhaps we should revisit this later.

Reply:
I agree, this maybe this would be nullable as well according to the Transaction Service docs, but we can revisit it later as it might need adjustments in other parts of the code as well 🙂